llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -60673,3 +60673,20 @@ Align X86TargetLowering::getPrefLoopAlignment(MachineLoop *ML) const {
     return Align(1ULL << ExperimentalPrefInnermostLoopAlignment);
   return TargetLowering::getPrefLoopAlignment();
 }
+
+bool X86TargetLowering::isDesirableToCommuteWithShift(
```

Comment:
remove this? it looks like it matched the generic implementation

Reply:
done